packages/react/vite.config.ts

```diff
@@ -8,12 +8,16 @@ export default defineConfig({
 		chunkSizeWarningLimit: 600,
 		lib: {
 			entry: 'src/index.ts',
-			formats: ['es']
```

Comment:
This was the main problem. The default value is ['es', 'umd'] so omitting it restores the UMD build.